libr/util/str.c

```diff
@@ -2951,7 +2951,7 @@ R_API char *r_str_from_ut64(ut64 val) {
 	if (!str) {
 		return NULL;
 	}
-	while (i < 8 && *v && *v >= '!' && *v <= '~') {
+	while (i < 8 && *v) {
```

Comment:
uhm, isnt this filtering causing more false positives?

Reply:
oh well you are checking this in the ds_print_ptr instead..

Reply:
yes

Reply:
`r_str_from_ut64` should just return a string and not do any filtering on it